scripts/auto-rebase/rebase.sh

```diff
@@ -166,16 +172,146 @@ update_lvms_images(){
 update_lvms_manifests() {
     title "Modifying LVMS manifests"
 
-    local dest="${REPOROOT}/assets/components/lvms"
-    local namespace_scoped_manifests=(
-        "topolvm-controller_v1_serviceaccount.yaml"
-        "topolvm-csi-provisioner_rbac.authorization.k8s.io_v1_role.yaml"
-        "topolvm-csi-provisioner_rbac.authorization.k8s.io_v1_rolebinding.yaml"
-        "topolvm-node_v1_serviceaccount.yaml"
-    )
-    for m in "${namespace_scoped_manifests[@]}"; do
-        yq -i '.metadata.namespace = "openshift-storage"' "${dest}/${m}" || return 1
-    done
+}
```

Comment:
It looks like `update_lvms_manifests` is no longer useful.

Reply:
Agreed, wanted to verify that rebase procedure does not need additional changes of namespaces after rebase is successful (see above comment on why I dont have the rebase running yet).